src/main/java/duke/Command.java

```diff
@@ -0,0 +1,203 @@
+package duke;
+
+import java.time.format.DateTimeParseException;
+import java.util.List;
+
+import duke.resource.TaskList;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+import duke.util.DukeException;
+
+
+/**
+ * Command class that handles the logic flow of commands input.
+ */
+
+public class Command {
+
+    private final String[] input;
+    private boolean shouldExit = false;
+
+    /**
+     * Constructor that creates a Command object.
+     * @param input command string that has been split
+     */
+
+    public Command(String[] input) {
+        this.input = input;
+    }
+
+    /**
+     * Evaluates the syntax of the input command and executes it, throwing
+     * a DukeException if the syntax is incorrect.
+     * @param tasks the TaskList object associated with the current Duke object
+     * @param ui the Ui object associated with the current Duke object
+     * @param storage the Storage object associated with the current Duke object
+     * @throws DukeException thrown if any errors are detected with the input
+     */
```

Comment:
Based on coding standard, the Javadoc comments should have empty line between description and parameter section and there needs to be punctuation (a full stop) behind each parameter description.

Reply:
Ah, I wasn't aware, thanks!